app/services/downstream_service.rb

```diff
@@ -18,6 +18,11 @@ def self.update_draft_content_store(downstream_payload)
       message = "Can only send draft, published and unpublished items to draft content store"
       raise DownstreamInvalidStateError.new(message)
     end
+    if downstream_payload.state != "draft" && draft_at_base_path?(downstream_payload.base_path)
+      message = "Can't send #{downstream_payload.state} item to draft content store, as there is a draft occupying the same base path"
+      raise DownstreamInvalidStateError.new(message)
```

Comment:
By "error situation" I meant that:
- It is an error to add a published item to the draft content store when there is a draft item occupying that base path

As for whether the worker should treat it as an error or an indiscretion, that's a reasonable argument but outside the context of this work. I'd be fine with them just alerting the logger.